### PR TITLE
Including documentation for Abandoned checkout API

### PR DESCRIPTION
**Pull request by ferrari-lucas**

### CODE REVIEW

**Review by gustavocgatto — APPROVED** [review]
Perfect!

File: resources/abandoned_checkout.md [hunk 1]
```diff
@@ -0,0 +1,627 @@
+
+Abandoned Checkout
+=====
+
+The abandoned checkout is created when the customer reaches the second checkout step with the cart, but for some reason it does not finish the process.
```
Comment (Nahpal) [inline resources/abandoned_checkout.md:5]: Change for:

"The abandoned checkout is created when the customer reaches checkout's seconds step, but for some reason it does not finish the process."

File: resources/abandoned_checkout.md [hunk 2]
```diff
@@ -0,0 +1,627 @@
...
+
+Notes
+----------
+> ***Note:*** The application must have `read_orders` or `write_orders` permission (depends on whether they just want to do GET or POST).
+
+> ***Note:*** A cart can be created up to `6 hours` after the purchase has been abandoned.
+
+> ***Note:*** The carts are accessible for `30 days` after being created.
+
+> ***Note:*** A purchase is only converted into a cart if the customer has arrived for the second checkout step.
```
Comment (Nahpal) [inline resources/abandoned_checkout.md:28]: Change for:

"A purchase is only converted into an abandoned cart if the customer has arrived to checkout's second step."

**Review by Nahpal — CHANGES_REQUESTED** [review]
Awesome work!

Just 2 small wordings to change i think

File: resources/abandoned_checkout.md (hunk 1 above)
Comment (ferrari-lucas) [reply resources/abandoned_checkout.md]: Resolved [here](https://github.com/TiendaNube/api-docs/pull/145/commits/33e228baa64813eb883514277cb3eae09cfeeeff)

File: resources/abandoned_checkout.md (hunk 2 above)
Comment (ferrari-lucas) [reply resources/abandoned_checkout.md]: Resolved [here](https://github.com/TiendaNube/api-docs/pull/145/commits/33e228baa64813eb883514277cb3eae09cfeeeff)

**Comment by heyBett** [comment]
Boa noite pessoal! Temos previsão para um webhook de abandoned_checkout? Os clientes pedem muito para recuperação de carrinho no meu app da RD Station.

**Review by Nahpal — APPROVED** [review]
Great!

**Pull request merged**